core/ui/src/main/kotlin/de/tum/informatics/www1/artemis/native_app/core/ui/material/colors/exercise_action_colors.kt

```diff
@@ -0,0 +1,13 @@
+package de.tum.informatics.www1.artemis.native_app.core.ui.material.colors
+
+import androidx.compose.foundation.isSystemInDarkTheme
+import androidx.compose.runtime.Composable
+import androidx.compose.ui.graphics.Color
+
+// Colors for the info message card in the exercise screen
+val skyBlueBackground: Color
```

Comment:
I'd prefer to have the color values wrapped inside a class / companion object, so that when accessing them its more readable eg `InfoMessageCardColors.text`
 
Same for the `difficulty_colors.kt` eg `DifficultyColors.easy`

Reply:
That actually makes a lot of sense! Good idea :)